src/components/RegistrationForm.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
ja bym ten plik utworzył w folderze Registration i wewnątrz index.ts, RegistrationForm.tsx, RegistrationFrom.interface.ts i jak pojawi się styled components to pewnie pojawi się plik Registration.style.ts czy coś takiego, również jak zostanie dodany cypress to będzie Registration.test.tsx 

sporo tego xD

Reply:
fixed